core/src/test/java/io/questdb/test/ServerMainShowPartitionsTest.java

```diff
@@ -121,7 +120,7 @@ public void testServerMainShowPartitions() throws Exception {
                 TableToken tableToken = createPopulateTable(cairoConfig, engine, defaultCompiler, defaultContext, tableName);
                 engine.awaitTable(tableName, 30, TimeUnit.SECONDS);
 
-                String finallyExpected = replaceSizeToMatchOS(EXPECTED, dbPath, tableToken.getTableName(), engine);
+                String finallyExpected = replaceSizeToMatchOS(EXPECTED, dbPath, tableToken.getTableName(), engine, new StringSink());
```

Comment:
@nwoolmer since I have moved the function `replaceSizeToMatchOS` to TestUtils as asked by @puzpuzpuz 
Earlier it was using `sink` from AbstractCairoTest when written in `ShowPartitionsTest`, so it didn't need it in params.